typings/index.d.ts

```diff
@@ -290,7 +279,7 @@ export interface ContextMessageUpdate extends Context {
    * @param ok  Specify True if everything is alright (goods are available, etc.) and the bot is ready to proceed with the order. Use False if there are any problems.
    * @param errorMessage Required if ok is False. Error message in human readable form that explains the reason for failure to proceed with the checkout (e.g. "Sorry, somebody just bought the last of our amazing black T-shirts while you were busy filling out your payment details. Please choose a different color or garment!"). Telegram will display this message to the user.
    */
-  answerPreCheckoutQuery(ok: boolean, errorMessage?: string): Promise<boolean>
+  answerPreCheckoutQuery (ok: boolean, errorMessage?: string): Promise<boolean>
```

Comment:
There is an error here. There shouldn't be a space.

Also, this file had been generated. There is no need to edit it) It should be generated correctly!